packages/manager/modules/pci/src/projects/project/ai/apps/apps.html

```diff
@@ -3,7 +3,14 @@
     data-type="warning"
     data-dismissable
 >
-    <span data-ng-bind="$ctrl.betaWarning"></span>
+    <p data-ng-bind="$ctrl.betaWarning" class="mb-1"></p>
+    <span data-translate="pci_ai_app_beta_warning_special_conditions"></span>
+    <a
+        data-ng-href="{{$ctrl.getSpecialConfitionsLink()}}"
```

Comment:
```suggestion
        data-ng-href="{{$ctrl.getSpecialConditionsLink()}}"
```

Reply:
Might also add : 

```
data-ng-href="{{:: $ctrl.getSpecialConditionsLink()}}"
```

to avoid recomputing it when not necessary